libs/langgraph/src/graph/graph.ts

```diff
@@ -540,6 +544,187 @@ export class CompiledGraph<
   /**
    * Returns a drawable representation of the computation graph.
    */
+  override async getGraphAsync(
+    config?: RunnableConfig & { xray?: boolean | number }
+  ): Promise<DrawableGraph> {
+    const xray = config?.xray;
+    const graph = new DrawableGraph();
+    const startNodes: Record<string, DrawableGraphNode> = {
+      [START]: graph.addNode(
+        {
+          schema: z.any(),
+        },
+        START
+      ),
+    };
+    const endNodes: Record<string, DrawableGraphNode> = {};
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let subgraphs: Record<string, CompiledGraph<any>> = {};
+    if (xray) {
+      subgraphs = Object.fromEntries(
+        (await gatherIterator(this.getSubgraphsAsync())).filter(
```

Comment:
is gather iterator just Array.from?

Reply:
`Array.from` doesn't work with async iterators

There is this but Node 22 only:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/fromAsync